docs/modules/ROOT/pages/migration_guide.adoc

```diff
@@ -73,37 +72,37 @@ components:
        reference: https://.../mongo.yaml
 ----
 
-Details can be found in the https://github.com/che-incubator/devworkspace-api/issues/4[corresponding issue].
+Details can be found in the https://github.com/devfile/api/issues/4[corresponding issue].
```

Comment:
All of the _"Details can be found in the >corresponding issue<."_ sentences would look better with:

* No passive tense
* Title of the linked issue

E.g.: _"For more details, see >Polymorphic component type UX in a devfile<."_